store/postgres/src/subgraph_store.rs

```diff
@@ -1351,6 +1351,23 @@ impl SubgraphStoreTrait for SubgraphStore {
         })
     }
 
+    fn drop_subgraph(&self, hash: &DeploymentHash) -> Result<(), StoreError> {
+        let deployments = self.subgraphs_for_deployment_hash(hash)?;
+        for (deployment, _) in deployments.into_iter() {
+            self.remove_subgraph(
+                SubgraphName::new(deployment.clone())
+                    .map_err(|_| StoreError::DeploymentNotFound(deployment))?,
+            )?;
```

Comment:
Dont this delete both the pending and current version of a subgraph that belongs to the same name as `hash` provided? Thought we are moving away from that behaviour?